Jan 29 Release Notes Updates (#602)
Axway/amplify-central: content/en/docs/amplify_relnotes/2025jan_amplify.md

---

## January 29, 2025

New features, enhancements, and bug fixes for the January 29 update.

### Marketplace updates for January 29, 2025

> * **Product Plan Visibility**

Reviewer: ```suggestion
* **Product plan visibility**
```

  (PRODUCT FOUNDRY, PLANS, PROVIDER EXPERIENCE, NEW FEATURE)<br />

>   Producers can now restrict the visibility of specific product plans by Platform Teams and / or Consumer Organizations, enabling tailored pricing and subscription models for different partners. This allows for more control over who can access free or paid plans based on you organization type. All newly created plans remain public unless explicitly restricted.

Reviewer: ```suggestion
  Providers can now restrict the visibility of specific product plans by Platform Teams and / or Consumer Organizations, enabling tailored pricing and subscription models for different partners. This allows for more control over who can access free or paid plans based on organization type. All newly created plans remain public unless explicitly restricted.
```

> * **Documentation Search Bar in Marketplace**

Reviewer: ```suggestion
* **Documentation search bar in Marketplace**
```

  (MARKETPLACE, CONSUMER EXPERIENCE, ENHANCEMENT, DOCUMENTATION)<br />

>   We have added a search bar to the product documentation navigation panel in the marketplace, making it easier for users to quickly find the content they need. You can search for topics, sections or articles by name in the documentation browse panel.

Reviewer: ```suggestion
  A search bar has been added to the Product Documentation navigation panel in the Marketplace, making it easier to quickly find content. You can search for topics, sections and articles by name in the documentation browse panel.
```
